javascript/code-challenges/challenges-10.test.js

```diff
@@ -26,7 +26,18 @@ For example:
 return: 23
 ------------------------------------------------------------------------------------------------ */
 const findMax = (matrix) => {
-  // Solution code here...
+  
+  let value = 0;
+  for( let i=0; i < matrix.length; i++) {
+    for (let j = 0; j<matrix[i].length; j++) {
+      if(matrix[i][j] > value) {
+        value = matrix[i][j];
+
+      }
+
+      }
+  }
+  return value;
 };
```

Comment:
Same here, nice work! I think when you have time, going back to incorporate map() and reduce() could be a good way to get some more practice in 👍